### PR TITLE
chore: bump LICENSE year

### PR DESCRIPTION
**Pull request by Syjalo**
**Please describe the changes this PR makes and why it should be merged:**
Bump LICENSE year

**Status and versioning classification:**

- This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Review by favna — CHANGES_REQUESTED** [review]
TL;DR: Very likely this won't be merged and you should stop this insane PR farming.

---

This is not how open source licenses work. You don't need to update the year at all. You're supposed to list the year of first publication.

https://opensource.stackexchange.com/a/5779

https://softwareengineering.stackexchange.com/a/130479

https://opensource.stackexchange.com/a/1523

In fact the way I understand it is that updating the license year is subsequently also damaging for your project because it implies that any new files added after the license change are licensed as of the current year, instead of as of the original publication date of the work.

Look at it this way. Is Mickey Mouse licensed as of 2022, or as of 1923. That's right, it's the latter. Open source software works the exact same.
